documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -61,3 +61,21 @@ To entirely ignore the display when connected, add the following line to `/boot/
 ----
 ignore_lcd=1
 ----
+
+== Attaching the Touch Display 2 LCD panel
```

Comment:
Perhaps the original "Attach the official 7-inch display" heading from the very top of this page ought to be updated to something like "Attaching the Touch Display LCD panel" ?

Reply:
done